core/src/04-feats/01-chapter-four.md

```diff
@@ -341,9 +341,17 @@ ___
 - **Cost:**  1 point
 - **Prerequisites:**
  - **Tier 1:** None
-- **Description:** Whether through brute strength or lightning reflexes, you are able to use the momentum of combat to maneuver around the battlefield with ease.
+- **Description:** If you have access to the *teleport* boon, you may use it instead of a normal move.
```

Comment:
You pasted the `Special` text over the `Description`. Original `Description` was:
> Whether through brute strength or lightning reflexes, you are able to use the momentum of combat to maneuver around the battlefield with ease.